webpack.config.js

```diff
@@ -21,7 +21,18 @@ module.exports = {
         use: {
           loader: "babel-loader"
         }
-      }
+      },
+      {
+        test:/\.css$/,
+        use:[
+          {loader: 'style-loader'},
+          {
+            loader: 'css-loader',
+            options: {
+                modules: true
```

Comment:
nitpick -- indentation is sorta weird here. I just realized we need to fix prettier in all the repos so that we don't have to worry about code styles

Reply:
added prettier and pretty quick to pre-commit in the next commit